app/views/simulator/edit.html.erb

```diff
@@ -714,6 +714,15 @@
   <% end %>
 </script>
 
+<!-- report an issue validator -->
+<script>
+  function validator() {
+    if (document.getElementById("issuetext").value.length == 0) {
+      alert("Description cannot be empty!")
```

Comment:
We are trying to remove all existing alerts from the simulator, can we use a inline message or a modal ?

Reply:
I guess an inline message would be good somewhat like we have in the login form.

Reply:
Okay , I will change the prompt to inline message .